cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list-header.tsx

```diff
@@ -105,9 +107,12 @@ function ObjectListHeader(props: Props): JSX.Element {
                                 <span style={{ marginLeft: 5 }}>Annotations filter</span>
                             </>
                         )}
-                        dropdownStyle={{ display: 'none' }}
                         onChange={changeAnnotationsFilters}
-                    />
+                    >
+                        {annotationsFiltersHistory.map((element: string): JSX.Element => (
+                            <Select.Option key={element} value={element}>{element}</Select.Option>
```

Comment:
Let's add comment here.
Something like: ``it is guaranteed, that each filter is unique``
because keys must be unique